app/scripts/metamask-controller.js

```diff
@@ -1286,7 +1286,6 @@ export default class MetamaskController extends EventEmitter {
         name: 'SignatureController',
         allowedActions: [
           `${this.approvalController.name}:addRequest`,
-          `${this.approvalController.name}:acceptRequest`,
           `${this.approvalController.name}:rejectRequest`,
```

Comment:
Do we still need the reject inside the controller?

Reply:
Removed.